src/docs/references/extended-data/index.md

```diff
@@ -22,11 +22,9 @@ extended data for some value types.
 See the [API reference](/references/api/) for each resource for
 information on supported extended data.
 
-There are five types of extended data, which can be divided into two
-groups - _data_ and _metadata_. _Data_ is written by users via the
-Marketplace API, while _metadata_ is written by operators via the Flex
-Console. Each type has different access semantics (i.e. who is allowed
-to read or write the data).
+There are four types of extended data, which are _Metadata_, _Private_,
```

Comment:
`Metadata` is mentioned twice => Public?

Reply:
Good catch! fixed!

Reply:
The words "public", "private" and "protected" don't make much sense alone. I think it should always be written as "public data", "private data" and "protected data". I don't think they need to be capitalized in the text either. So my suggestion is that this paragraph becomes:

```
There are four types of extended data: _metadata_, _private data_, _protected data_ 
and _public data_. Each type has different access semantics (i.e. who is allowed
to read or write the data).
```